source/server/server.cc

```diff
@@ -449,7 +449,11 @@ void InstanceImpl::initialize(const Options& options,
       stats().symbolTable(), bootstrap_.node(), bootstrap_.node_context_params(), local_address,
       options.serviceZone(), options.serviceClusterName(), options.serviceNodeName());
 
-  Configuration::InitialImpl initial_config(bootstrap_, options, *this);
+  Configuration::InitialImpl initial_config(bootstrap_, options);
+  runtime_singleton_ = std::make_unique<Runtime::ScopedLoaderSingleton>(
+      component_factory.createRuntime(*this, initial_config));
+  initial_config.initAccesslog(bootstrap_, *this);
+  hooks.onRuntimeCreated();
```

Comment:
I don't think this matters, but just in case: @alyssawilk -- was there an assumption that a listener manager would be instantiated before `onRuntimeCreated()` can be called?

Reply:
ok, i move “hooks.onRuntimeCreated()” to the origin place